game/CMakeLists.txt

```diff
@@ -311,6 +311,12 @@ if(WIN32)
 	set_target_properties(performous PROPERTIES OUTPUT_NAME "Performous")
 endif()
 
+if(WIN32 AND MSVC)
+	target_compile_options(performous PUBLIC /WX)
+else()
+	target_compile_options(performous PUBLIC -Werror)
```

Comment:
just wondering if this also works for clang?
If yes let's go!

Reply:
Might be better to just heck for MSVC isntead of WIN32, as MSVC _can_ be used on unix....

Reply:
> just wondering if this also works for clang? If yes let's go!

It should cause clang uses gccs command line AFAIK

Reply:
> Might be better to just heck for MSVC isntead of WIN32, as MSVC _can_ be used on unix right?

Not that I know. msvc is visual studio only. There is a mac version, but it only supports .net and no c++ AFAIK.

Reply:
👍